### PR TITLE
Api routes

### PR DESCRIPTION
**Pull request by acres42**
Backend API calls functioning as expected

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const db = require('../database-mongodb/index.js');
+const Product = require('../database-mongodb/Product.js');
+
+const app = express();
+const PORT = 3002;
+
+
+app.use(express.static(__dirname + '/../client/dist'));
```
Comment (ascherj) [inline server/index.js]: Here's a suggestion that eslint had me change after I configured it to use the Airbnb style guide. You'll also need to require the `path` module at the top of your file.

```suggestion
app.use(express.static(path.join(__dirname, '/../client/dist')));
```

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const db = require('../database-mongodb/index.js');
+const Product = require('../database-mongodb/Product.js');
+
```
Comment (ascherj) [inline server/index.js]:
```suggestion
const path = require('path');
```

**Review by ascherj — COMMENTED** [review]
Looking good so far @acres42. I would suggest looking into configuring ESLint and Jest before diving into your implementation to ensure good coding practices from the start.

Also, does "pav" stand for "people also viewed"?

**Comment by ascherj** [comment]
You'll also want to test your API to check if your endpoints correctly retrieve the desired info from the database. [SuperTest](https://github.com/visionmedia/supertest) is a great library for that.

**Comment by acres42** [comment]
@ascherj : GET OUT OF MY HEAD JAKE! lol. my next step is adding jest and maybe super test. i am using POSTMAN and everything is coming back from the server as expected. I will look into it though. Also, yes, PAV is the people also viewed. Thanks for the suggestions. Committed!

**Pull request merged**